Complete To Do's of new user guide (#184)
moderncv/moderncv: manual/moderncv_userguide.tex

  \quote{<quote>}
\end{lstlisting}

> \note{Long names and URLs can break some CVs and cover letters depending on the used style. Inserting line breaks for names with \code{\\\\} can help for \texttt{oldstyle} or \texttt{fancy}. A second measure to mitigate issues with long URLs would be adjustments to lengths described in section \ref{subsection:adjusting:lengths}.}

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

> \note{Long names and URLs can break some CVs and cover letters depending on the used style. Inserting line breaks for names with \code{\\\\} can help for \texttt{oldstyle} or \texttt{fancy}. A second measure to mitigate issues with long URLs would be adjustments to lengths described in section \ref{subsection:adjusting:lengths}.}

Author: Ah, didn´t know about that, but I agree:
Maybe we should open a separate issue?

\paragraph{Pictures.}

> For some occasions it is needed to include pictures besides the portrait photo. For this task it is helpful to know basic lengths used in \moderncv. For a more detailed description of lengths see section \ref{subsection:adjusting:lengths}.

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

> Pictures can be included i. e. with \code{\\includegraphics}.

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

\subsection{Letter of motivation}

In order to provide additional information for a job application {\moderncv} provides a motivation letter.\newline

> Define the following recipient data fields to customize your letter.

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

\paragraph{\code{\\recipient}}
A command for the letter's recipient. This allows line breaks to properly format the recipient's address.
\begin{lstlisting}
  \recipient{<Team or contact person>}{<company>\\<street>\\<city>}
\end{lstlisting}

\paragraph{\code{\\date}}
A command to set the letter's date.
\begin{lstlisting}
  \date{<your custom date>}
\end{lstlisting}

\paragraph{\code{\\subject}}
A command to define the job application subject.
\begin{lstlisting}
  \subject{<subject>}
\end{lstlisting}

\paragraph{\code{\\opening}}
A command to set your greeting formula.
\begin{lstlisting}
  \opening{<greeting formula>}
\end{lstlisting}

\paragraph{\code{\\closing}}
A command to set your closing formula.
\begin{lstlisting}
  \closing{<closing formula>}

> \end{lstlisting}

Author: hehe, you are right. Better do not translate things from your native language directly... ^^
Thanks for correcting this!
Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

\paragraph{\code{\\signature}}
An optional command to include your signature after the closing part. For this feature the \code{\\includegraphics} macro is used. The first argument defines the scale. The second argument holds the filename of your scanned signature.
\begin{lstlisting}
  \signature{<scale>}{<signature filename>}

> \end{lstlisting}

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

\paragraph{\code{\\enclosure}}

> A command to list the attached documents. The first optional parameter allows a re-definition of the default ``Enclosure'' label.

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

  \renewcommand*{\fixedphonesymbol}{\ding{38}~}
\end{lstlisting}

\paragraph{Alternative to icons}

> In case \texttt{symbols} (default) are not wished there is also the option to use \texttt{letters} instead. This is defined by the macro \code{\\moderncvicons}. It must be called after setting the style in the preamble.

Reviewer: ```suggestion
In case you do not want \texttt{symbols} (default) for the icons, there is also the option to replace icons with \texttt{letters} instead. This is set by the macro \code{\\moderncvicons}, which must be called after setting the style in the preamble.
```

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

\begin{lstlisting}
  \moderncvicons{<symbols|letters>}
\end{lstlisting}

> This mechanism is also used to determine the icon set. In principle the \code{\\moderncvicons} can be used to determine the icon set with the possible values \texttt{marvosym}, \texttt{awesome} or \texttt{academic}. But due to different \Latex compiler support it is recommended to let {\moderncv} decide the correct icon set.

Reviewer: ```suggestion
This mechanism is also used to determine the icon set. In principle, the \code{\\moderncvicons} can be used to determine the icon set with the possible values \texttt{marvosym}, \texttt{awesome} or \texttt{academic}, but it is recommended to let {\moderncv} decide the correct icon set due to differing \Latex compiler support.
```

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

One should be careful though, as the length is normally calculated to avoid any overlap with the personal information.
This should be used at one's own typographical risk.

> The spacing between the hints column and the CV entries can be adjusted by setting \code{\\separatorcolumnwidth}. The default value in units of \code{\\textwidth} is $0.025$.

Reviewer: ```suggestion
The spacing between the hints column and the CV entries can be adjusted by setting\\\code{\\separatorcolumnwidth}. The default value is \code{0.025\\textwidth}.
```
The line exceeds the margin in the PDF, so I added a manual line break.

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

\begin{lstlisting}
  \setlength{\separatorcolumnwidth}{0.05\textwidth}
\end{lstlisting}

> Adjusting the width of the footer can be achieved with \code{\\footwidth}. Due to internal calculations \code{\\renewcommand} is needed. The default value is $0.8$ of \code{\\textwidth}.

Reviewer: ```suggestion
The width of the footer can be adjusted by setting \code{\\footwidth}. Due to internal calculations \code{\\renewcommand} is needed. The default value is \code{0.8\\textwidth}.
```

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

\begin{lstlisting}
  \renewcommand*{\recomputefootlengths}{%
  \setlength{\footwidth}{0.9\textwidth}}
\end{lstlisting}

> The width of the quote can be adjusted by setting \code{\\quotewidth}. Due to internal calculations \code{\\renewcommand} is needed. The default value is $0.65$ of \code{\\textwidth}.

Reviewer: ```suggestion
The width of the quote can be adjusted by setting \code{\\quotewidth}. Due to internal calculations \code{\\renewcommand} is needed. The default value is \code{0.65\\textwidth}.
```

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

\end{lstlisting}
\code{<width>} and \code{<exp_width>} should be lengths smaller than \code{\\textwidth}, and \code{<factor>} must be between 0 and 1.

\subsection{Additional link commands}%% adapted from Cristina Sambo's documentation

> To create textlinks to a website or a email address, use the following commands:
> A general weblink with an optional text.  
> \begin{lstlisting}
>   \link[<text>]{<link>}
>   %% example
>   \link[name of the link]{ftp://ftp.somesite.org}
> \end{lstlisting}

Reviewer: ```suggestion
To create links to a website or a email address, use the following commands:

A general weblink with optional text.  
\begin{lstlisting}
  \link[<text>]{<link>}
  %% example
  \link[name of the link]{ftp://ftp.somesite.org}
\end{lstlisting}
```

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

> A http link. The http prefix is generated automatically and is not needed in the link argument.
> \begin{lstlisting}
>   \link[<text>]{<link>}
>   %% example
>   \link[goto http side]{www.somehttpsite.org}
> \end{lstlisting}

Reviewer: ```suggestion
An HTTP link. The HTTP prefix is generated automatically and is not needed in the link argument.
\begin{lstlisting}
  \link[<text>]{<link>}
  %% example
  \link[goto HTTP site]{www.somehttpsite.org}
\end{lstlisting}
```

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

> A https link. The https prefix is generated automatically and is not needed in the link argument.
> \begin{lstlisting}
>   \link[<text>]{<link>}
>   %% example
>   \httpslink[goto https site]{www.somesecuresite.org}
> \end{lstlisting}

Reviewer: ```suggestion
An HTTPS link. The HTTPS prefix is generated automatically and is not needed in the link argument.
\begin{lstlisting}
  \link[<text>]{<link>}
  %% example
  \httpslink[goto HTTPS site]{www.somesecuresite.org}
\end{lstlisting}
```

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

> A e-mail link. The mailto prefix is generated automatically and is not needed in the link argument.

Reviewer: ```suggestion
An email link. The mailto prefix is generated automatically and is not needed in the link argument.
```
"email" rather than "e-mail" is consistently used throughout the rest of the doc

\label{section:implementationDetails}

\subsection{Creating your own styles}

> Currently {\moderncv} ships with five differnt styles which are \texttt{casual} as default, \texttt{classic}, \texttt{banking}, \texttt{oldstyle} and \texttt{fancy}. All styles except \texttt{oldstyle} and \texttt{fancy} possess additional option parameters which were described in section \ref{section:customization:stylesAndOptions}.
> The numbering scheme for the above named variants is in that order $1$, $2$, $3$, $4$ and $5$.

Reviewer: ```suggestion
Currently {\moderncv} ships with five different styles: \texttt{casual} (default), \texttt{classic}, \texttt{banking}, \texttt{oldstyle} and \texttt{fancy}. Internally, these styles are numbered 1, 2, 3, 4 and 5, respectively. All styles except \texttt{oldstyle} and \texttt{fancy} possess additional option parameters which were described in section \ref{section:customization:stylesAndOptions}.
```

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

> To achive a whole new variety it is possible to combine different styles' head, body and foot with each other. This is done with the command \code{\\moderncvhead}, \code{\\moderncvbody} and \code{\\moderncvfoot} in the preamble. Here is an example of a mixture between the banking head, oldstyle body and the foot:

Reviewer: ```suggestion
Different styles' head, body and foot can be combined to create custom styles. This is done by setting \code{\\moderncvhead}, \code{\\moderncvbody} and \code{\\moderncvfoot} in the preamble. Here is an example of a custom style with a \textt{banking} head, an \textt{oldstyle} body and a \textt{casual} foot:
```

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

\begin{lstlisting}
  \moderncvhead{3}
  \moderncvbody{4}
  \moderncvfoot{1}
\end{lstlisting}

> Furthermore options for {\moderncv} parts can be set as well in such combinations. To get a CV with a classic head and fullrule body define the following in the preamble:

Reviewer: ```suggestion
Furthermore, options for each part of {\moderncv} can be set as well. To get a CV with a \texttt{classic} head and a \texttt{banking} body with full rules, define the following in the preamble:
```

Author: Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)

\label{section:implementationDetails}

\subsection{Creating your own styles}

> Currently {\moderncv} ships with five different styles: \texttt{casual} (default), \texttt{classic}, \texttt{banking}, \texttt{oldstyle} and \texttt{fancy}. Internally, these styles are numbered 1, 2, 3, 4 and 5, respectively. All styles except \texttt{oldstyle} and \texttt{fancy} possess additional option parameters which were described in section \ref{section:customization:stylesAndOptions}.

Reviewer 2: ```suggestion
Currently {\moderncv} ships with six different styles: \texttt{casual} (default), \texttt{classic}, \texttt{banking}, \texttt{oldstyle}, \texttt{fancy} and \texttt{contemporary}. Internally, these styles are numbered 1, 2, 3, 4, 5 and 6, respectively. All styles except \texttt{oldstyle} and \texttt{fancy} possess additional option parameters which were described in section \ref{section:customization:stylesAndOptions}.
```

Author: Ah, yes, in the meantime we already have six cool styles... Forgot that passage here, thanks for mentioning it!
Is now resolved with [e0e9259](https://github.com/moderncv/moderncv/pull/184/commits/e0e9259201620d26f233190f0d366627be653b81)

> ifferent styles' head, body and foot can be combined to create custom styles. This is done by setting \code{\\moderncvhead}, \code{\\moderncvbody} and \code{\\moderncvfoot} in the preamble. Here is an example of a custom style with a \texttt{banking} head, an \texttt{oldstyle} body and a \texttt{casual} foot:

Reviewer 2: ```suggestion
Different styles' head, body and foot can be combined to create custom styles. This is done by setting \code{\\moderncvhead}, \code{\\moderncvbody} and \code{\\moderncvfoot} in the preamble. Here is an example of a custom style with a \texttt{banking} head, an \texttt{oldstyle} body and a \texttt{casual} foot:
```

Author: Hmm... Somebody stole a `D` here. Thanks! Is now resolved with [e0e9259](https://github.com/moderncv/moderncv/pull/184/commits/e0e9259201620d26f233190f0d366627be653b81).
